src/SpectatorView.Unity/Assets/PhotoCapture/Microsoft.MixedReality.PhotoCapture.msb4u.csproj.meta

```diff
@@ -0,0 +1,12 @@
+fileFormatVersion: 2
+guid: ef3a3e4df5c774c4db3673795871a675
+ScriptedImporter:
+  fileIDToRecycleName:
+    11400000: Microsoft.MixedReality.PhotoCapture.msb4u
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
+  script: {fileID: 11500000, guid: 0e4dca1acba66fd478b3854f09a32ec7, type: 3}
+  buildEngine: 0
+  profiles: []
```

Comment:
Should these new meta files exist? Seems like something that might be missing from the .gitignore since I don't see the corresponding files, just the meta files.

Reply:
I'm not sure what the call is on this. I'll ask andrei today.

Reply:
we want to include these files, there may be something wrong if we aren't including csprojs .I will follow up in a different review.